data/config.dist.json

```diff
@@ -2,7 +2,8 @@
   "symbol-whitelist" : [
     "null", "true", "false",
     "static", "self", "parent",
-    "array", "string", "int", "float", "bool", "iterable", "callable", "void", "object"
+    "array", "string", "int", "float", "bool", "iterable", "callable", "void", "object",
+    "Composer\\InstalledVersions"
```

Comment:
Hmm, don't think it should be ignored completely - it should be whitelisted if `composer-runtime-api:^2` is required by the package.